<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MAINRUNTEST</class>
 <widget class="QMainWindow" name="MAINRUNTEST">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuDataStream">
    <property name="title">
     <string>DataStream</string>
    </property>
    <addaction name="actionClear"/>
    <addaction name="actionLocked"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDataStream"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="toolTip">
    <string>Save the collected data to a specified file.</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionLocked">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Locked</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
